feature/platform/tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -380,8 +380,8 @@ func TestSwitchChip(t *testing.T) {
 
 		cName := card.GetName()
 		t.Run(fmt.Sprintf("Backplane:%s", cName), func(t *testing.T) {
-			if deviations.BackplaneFacingCapacityUnsupported(dut) && regexp.MustCompile("NPU[0-9]$").Match([]byte(card.GetName())) {
```

Comment:
For Juniper, the regexp that matches certain nodes by name is needed along with the deviation flag check. Instead of removing the regexp, could you please do something like below:

if deviations.BackplaneFacingCapacityUnsupported(dut) {
	if (dut.Vendor() == ondatra.NOKIA) || (dut.Vendor() == ondatra.JUNIPER && regexp.MustCompile("NPU[0-9]$").Match([]byte(card.GetName()))) {
	...
	...
	}
}

Thanks!

Reply:
updated